x-pack/plugins/cloud_security_posture/server/routes/benchmark_rules/get_states/v1.ts

```diff
@@ -43,3 +47,32 @@ export const getCspBenchmarkRulesStatesHandler = async (
     );
   }
 };
+
+export const buildMutedRulesFilter = async (
+  encryptedSoClient: ISavedObjectsRepository
+): Promise<QueryDslQueryContainer[]> => {
+  const rulesStates = await getCspBenchmarkRulesStatesHandler(encryptedSoClient);
+
+  const mustNotFilter = [];
```

Comment:
nit: the naming here is a bit confusing, as we call it `mustNot` but what is generated is a `must` clause. Yes, what's returned from this function ends up in the `must_not` query, but these are quite distant from each other. `mutedRulesFilterQuery` would be more clear. We can also get rid of `mustNotClause` for the same reason and maybe inline it to avoid finding the right name for it.